src/boot/efi/chipid-devices.h

```diff
@@ -0,0 +1,223 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#include "efi.h"
+
+struct device {
+        const char16_t Name[128];
+        const char Compatible[128];
+        EFI_GUID Ids[32];
+};
+
+static struct device devices[] = {
+        {
+                .Name = L"Acer Aspire 1",
+                .Compatible = "acer,aspire1",
```

Comment:
Ah, now I see: you have a built-in translation table for SMBIOS values to DT compatible strings. So we will never compare apples (SMBIOS) to onions (DT compatibles), right?

And at least `acer,aspire1` is even an officially recognized compatible. The others as well?

Reply:
Yeah, was now also wondering if it is based to maintain this table in the code or rather allow some UKI build-time definitions [as well].